client/src/App.js

```diff
@@ -5,7 +5,7 @@ import './App.css';
 import Header from './components/Header';
 import Launchpad from './pages/Launchpad';
 import LogExercise from './pages/LogExercise';
-import AppContext from './context/AppContext';
+import { AppContext } from './context/AppContext';
```

Comment:
Ideally, you wouldn't need to import AppContext. Instead, `./context/AppContext` would export a Provider, say, `AppProvider`.